.github/workflows/cd-executor.yml

```diff
@@ -1,60 +1,115 @@
 name: Publish Release
 
 on:
-    workflow_call:
-      inputs:
-        service-name:
-          description: 'Service name'
-          type: string
-          required: true
-        working-dir:
-          description: 'Working directory'
-          type: string
-          required: true
+  workflow_call:
+    inputs:
+      working-dir:
+        description: "Working directory"
+        type: string
+        required: true
 
 jobs:
   release:
     runs-on: ubuntu-latest
 
     steps:
-     - name: Checkout code
-       uses: actions/checkout@v3
-       with:
-        ref: ${{ github.event.inputs.target-branch }}
-
-     - name: Run ballerina build
-       uses: ballerina-platform/ballerina-action@2201.8.4
-       with:
-        args:
-          pack
-       env:
-          WORKING_DIR:  ${{ inputs.working-dir }}
-          JAVA_HOME: /usr/lib/jvm/default-jvm   
-
-     - name: Get tag from Ballerina.toml and create tag name
-       working-directory: ${{ inputs.working-dir }}
-       run: |
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run ballerina build
+        uses: ballerina-platform/ballerina-action@2201.8.4
+        with:
+          args: pack
+        env:
+          working-dir: ${{ inputs.working-dir }}
```

Comment:
for bal build we need to use WORKING_DIR

Reply:
fixed with [6d9bf5b](https://github.com/wso2/open-healthcare-prebuilt-services/pull/45/commits/6d9bf5b2891bd6c99c43d27980ec381215719a9d)